src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -1,20 +1,346 @@
+
+Newly introduced endpoints:
+* `POST /api/tracker`
+* `GET /api/enrollments`
+* `GET /api/events`
+* `GET /api/trackedEntities`
+* `GET /api/relationships`
+
+Significant changes occurred in version 2.36 to make interface with clients homogeneous, and to allow more flexible ways to use the services.
+
+> ***NOTE***
+> 
+> The old endpoints are marked as deprecated, but still work.<br>
+>Not all the functionalities are ready in the new endpoints yet.
+
+
+> ***NOTE 2***
+>
+> These endpoints only support `JSON` format as input/output 
+
+## Changes in interface
+
+Most input/output entities have been adapted to use consistent naming across all services
+
+### Tracker Import changelog (`POST`)
+
+This table highlights V2.36 differences in import payload compared to V2.35
+
+|Entity|V2.35|V2.36|
+|---|---|---|
+|**Attribute**|`created`<br>`lastUpdated`|`createdAt`<br>`updatedAt`|
+|**DataValue**|`created`<br>`lastUpdated`|`createdAt`<br>`updatedAt`|
+|**Enrollment**|`enrollment`<br>`enrollment`<br>`created`<br>`createdAtClient`<br>`lastUpdated`<br>`lastUpdatedAtClient`<br>`trackedEntityInstance`<br>`enrollmentDate`<br>`incidentDate`<br>`completedDate`|`uid`<br>`enrollment`<br>`createdAt`<br>`createdAtClient`<br>`updatedAt`<br>`updatedAtClient`<br>`trackedEntity`<br>`enrolledAt`<br>`occurredAt`<br>`completedAt`|
+|**Event**|`trackedEntityInstance`<br>`eventDate`<br>`dueDate`<br>`created`<br>`createdAtClient`<br>`lastUpdated`<br>`lastUpdatedAtClient`<br>`completedDate`|`trackedEntity`<br>`occurredAt`<br>`scheduledAt`<br>`createdAt`<br>`createdAtClient`<br>`updatedAt`<br>`updatedAtClient`<br>`completedAt`|
+|**Note**|`storedDate`<br>`lastUpdated`|`storedAt`<br>`updatedAt`|
+|**ProgramOwner**|`ownerOrgUnit`<br>`trackedEntityInstance`|`orgUnit`<br>`trackedEntity`|
+|**RelationshipItem**|`trackedEntityInstance.trackedEntityInstance`<br>`enrollment.enrollment`<br>`event.event`|`trackedEntity`<br>`enrollment`<br>`event`|
+|**Relationship**|`created`<br>`lastUpdated`|`createdAt`<br>`updatedAt`|
+|**TrackedEntity**|`trackedEntityInstance`<br>`trackedEntityInstance`<br>`created`<br>`createdAtClient`<br>`lastUpdated`<br>`lastUpdatedAtClient`|`uid`<br>`trackedEntity`<br>`createdAt`<br>`createdAtClient`<br>`updatedAt`<br>`updatedAtClient`|
+
+### Tracker Export changelog (`GET`)
+
+`GET` endpoint responses format respect the naming convention reported in the previous paragraph, however, some changes were made regarding
+input parameter to respect the same naming conventions.
+
+These tables highlight V2.36 differences in input parameters for `GET` endpoints compared to V2.35
+
+#### Input parameters changes for `GET /api/tracker/enrollments`
+|V2.35|v2.36|
+|---|---|
+|`ou`|`orgUnit`|
+|`lastUpdated`|`lastUpdateDuration`|
+|`updatedAfter`|`updatedWithin`|
+|`programStartDate`|`programEndDate`|
```

Comment:
This parameter changed just the name or it is a different parameter?

Reply:
good catch, it's just bad table formatting, thank you!